doc/nrf/ug_thread_configuring.rst

```diff
@@ -123,30 +123,6 @@ You can change the stack version by using the following Kconfig options:
   This option is enabled by default if no other option is selected.
 * :kconfig:`CONFIG_OPENTHREAD_THREAD_VERSION_1_2` - Selects the Thread stack version that is compliant with the Thread 1.2 Specification.
 
-By selecting support for Thread 1.2, you enable the following features in addition to the :ref:`Thread 1.1 features <thread_ug_supported_features>`:
```

Comment:
I believe we should keep this information for customers.

Reply:
restored.